pkg/frontend/nvme_controller.go

```diff
@@ -95,15 +95,6 @@ func (s *Server) CreateNvmeController(_ context.Context, in *pb.CreateNvmeContro
 		log.Printf("error: %v", err)
 		return nil, err
 	}
-	// check input parameters validity
-	if in.NvmeController.Spec == nil || in.NvmeController.Spec.SubsystemNameRef == "" {
-		return nil, status.Error(codes.InvalidArgument, "invalid input subsystem parameters")
-	}
-	// Validate that a resource name conforms to the restrictions outlined in AIP-122.
-	if err := resourcename.Validate(in.NvmeController.Spec.SubsystemNameRef); err != nil {
```

Comment:
should this be replaced with `in.Parent` and not completely removed ?

Reply:
resloved